metricbeat/metricbeat.full.yml

```diff
@@ -953,6 +953,9 @@ output.elasticsearch:
   # default is 7 files.
   #number_of_files: 7
 
+  # Permissions to use for file creation. The default is 0600.
+  #permissions: 0666
```

Comment:
This file is generated based on [libbeat/_meta/config.reference.yml](https://github.com/elastic/beats/blob/d07f82d6dff6a7a8ee0ebcb166672ba1daf2a1cb/libbeat/_meta/config.reference.yml#L516-L536). After updating the file in libbeat you should run `make update` from the `beats` dir to re-generate the config files.